cluster/images/etcd/Makefile

```diff
@@ -62,7 +62,7 @@ endif
 # This option is for running docker manifest command
 export DOCKER_CLI_EXPERIMENTAL := enabled
 # golang version should match the golang version from https://github.com/coreos/etcd/releases for the current ETCD_VERSION.
```

Comment:
Just realized the golang version can be found from https://github.com/etcd-io/etcd/blob/master/CHANGELOG-3.4.md#go-1, instead of https://github.com/coreos/etcd/releases, could we update the comment?

Reply:
I actually couldn't find any info on golang version from https://github.com/coreos/etcd/releases. On the other hand,  etcd's changelog is manually updated, and it is not always guaranteed to have an entry for golang version for a patch release. The most reliable way would be to download the official etcd release and run `etcd --version`. I feel like we can update the comment when we update etcd version next time. I prefer we get this PR merged sooner so that we have longer soak time.

Reply:
SGTM.